week-4/A-objects-intro/exercise-part-1.js

```diff
@@ -6,4 +6,29 @@ can describe with a JavaScript object
 Assign each of them to a separate variable
 
 */
-
+var cars = {
+  brand: "BMW",
+  type: "sedan",
+  color: "white"
+};
+var tv = {
+  brand: "LG",
+  type: "smart 4k",
+  size: 60
+};
+var tables = {
+  type: "wood",
+  shape: "round",
+  size: 15 * 15

```

Comment:
That's a very good way of keeping your code readable and conveying that you're expressing the size of an area.